동현/12주차/2252.py

```diff
@@ -0,0 +1,31 @@
+import sys
+from collections import deque
+input = sys.stdin.readline
+
+N, M = map(int, input().split())
+graph = [[] for _ in range(N+1)]    # 인접노드를 넣을 2차원 배열 선언
+inDegree = [0]*(N+1)                # 차수 를 넣을 배열 선언
+
+for _ in range(M):
+    A, B = map(int, input().split())
+    graph[A].append(B)              # A에 대한 인접노드 B 추가
+    inDegree[B] += 1                # B 차수 증가
+
+q = deque()
```

Comment:
그냥 queue로 명명하시는게 좋아보여용, 한글자로 짓는건 임시변수 말고는 피해야한다고 생각합니다
```suggestion
queue = deque()
```